File: packages/android/src/main/kotlin/gov/civui/components/CivItemizedItem.kt
// CivUI — CivItemizedItem for Jetpack Compose
// Single label + amount row inside a `civ-itemized-total`. Mirrors
// civ-itemized-item.
//
// Placeholder body — the prop surface satisfies schema parity. See
// .claude/rules/audit-debt.md → "Native platform implementation pass".

package gov.civui.components

import androidx.compose.foundation.layout.Row
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier

@Composable
fun CivItemizedItem(
    label: String = "",
    amount: Double = 0.0,
    valueLabel: String = "",
    note: String = "",
    intent: String = "",
    modifier: Modifier = Modifier,
) {
    Row(modifier = modifier) {
        // Placeholder — see audit-debt.md for native implementation status.
    }
}

File: packages/ios/Sources/CivUI/CivItemizedItem.swift
// CivUI — CivItemizedItem for SwiftUI
// Single label + amount row inside a `civ-itemized-total`. Mirrors
// civ-itemized-item.
//
// Placeholder body — the prop surface satisfies schema parity. See
// .claude/rules/audit-debt.md → "Native platform implementation pass".

import SwiftUI

public struct CivItemizedItem: View {
    /// Row label (left side).
    public var label: String

    /// Numeric amount (right side). Formatted by the parent's currency / locale.
    public var amount: Double

    /// Escape hatch for non-numeric value text (e.g. `Pending`).
    public var valueLabel: String

    /// Optional secondary note rendered below the label.
    public var note: String

    /// Per-row tint. `''` | `positive` | `negative` | `neutral`.
    public var intent: String

    public init(
        label: String = "",
        amount: Double = 0,
        valueLabel: String = "",
        note: String = "",
        intent: String = ""
    ) {
        self.label = label
        self.amount = amount
        self.valueLabel = valueLabel
        self.note = note
        self.intent = intent
    }

    public var body: some View {
        EmptyView()
    }
}
